### PR TITLE
"instructions updated"

### PR DESCRIPTION
**Pull request by IMPranshu**
Signed-off-by: P.K. Agrawal <pranshukumar1999@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: Meshery-Adapters/consul-meshery-adapter/step1.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ Meshery can be downloaded, installed, and launched with a single command:
 
 Meshery is now available at host:`9081`.
 
-Find the **Meshery UI** tab is configured to be listening at `http://localhost:9081`/
+Find the **Meshery Server** tab is configured to be listening at `http://localhost:9081`/
```
Comment (asubedy) [inline Meshery-Adapters/consul-meshery-adapter/step1.md:17]: Please revert this change back we still want the instruction to state Meshery UI. The tab name should be changed to Meshery UI, which is a different issue to work on.

File: Meshery-Adapters/consul-meshery-adapter/step1.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ Meshery can be downloaded, installed, and launched with a single command:
...
 
-- Click on the **Meshery UI** tab
+- Click on the **Meshery Server** tab
```
Comment (asubedy) [inline Meshery-Adapters/consul-meshery-adapter/step1.md:19]: This too. Please change back to Meshery UI.

**Review by asubedy — CHANGES_REQUESTED** [review]

File: Meshery-Adapters/consul-meshery-adapter/step1.md (hunk 1 above)
Comment (IMPranshu) [reply Meshery-Adapters/consul-meshery-adapter/step1.md:17]: ok @asubedy

File: Meshery-Adapters/consul-meshery-adapter/step1.md (hunk 2 above)
Comment (IMPranshu) [reply Meshery-Adapters/consul-meshery-adapter/step1.md:19]: ok @asubedy

**Review by suhaniii — APPROVED** [review]

**Comment by asubedy** [comment]
@IMPranshu  Can you please also update the screenshots on the instructions? The current screenshot does not match the current Meshery UI.

**Comment by IMPranshu** [comment]
Yes, I am trying to do that but I am not able to sign in from yesterday.

**Comment by asubedy** [comment]
> Yes, I am trying to do that but I am not able to sign in from yesterday.

Are you not able to sign in on the lab too?

**Comment by IMPranshu** [comment]
I am not able to sign in to the labs. I shared this error on slack yesterday when I was trying to get into the labs for taking some screenshots. Anyways will try again soon to sign in

**Comment by IMPranshu** [comment]
Please see if it is ok and if I have to add something else to this.

**Comment by asubedy** [comment]
@IMPranshu Great work 🚀 . 
Please update the following screenshot also

- install-consul.png
- consul-installed.png
- consul-sample-app.png
- consul-imagehub-success.png

Also in `step2.md` at [`In the Consul Management Page`](https://github.com/layer5io/service-mesh-labs/blob/master/Meshery-Adapters/consul-meshery-adapter/step2.md#2-in-the-consul-management-page) section, its second point can be changed
- The card name is different
- And the option after clicking the `+`sign is different now, so change that too

In `step3.md` 
On the [`Expose Your Sample Application`](https://github.com/layer5io/service-mesh-labs/blob/master/Meshery-Adapters/consul-meshery-adapter/step3.md#expose-your-sample-application) section, `Linkerd` is being used so change that to `Consul`.

**Comment by IMPranshu** [comment]
Sure will do that shortly.

**Comment by IMPranshu** [comment]
Will be travelling for 2 days, I will continue to fix this after that @asubedy Will that be ok?

**Comment by asubedy** [comment]
All right @IMPranshu 👍

**Comment by IMPranshu** [comment]
@asubedy can we merge this PR, I am facing some issues while following the instructions. Will update the mentioned and Open a PR again. Will that be ok?

**Comment by leecalcote** [comment]
Gents, how are we proceeding? @IMPranshu its been a couple months. Have you completed the other changes?

**Comment by asubedy** [comment]
@leecalcote the PR can be merged. The update is not entirely complete, because we got blocked when Istio couldn't work properly on Katacoda.
